bot/exts/evergreen/fun.py

```diff
@@ -60,15 +60,17 @@ def __init__(self, bot: Bot) -> None:
     @commands.command()
     async def roll(self, ctx: Context, num_rolls: int = 1) -> None:
         """Outputs a number of random dice emotes (up to 6)."""
-        output = ""
-        if num_rolls > 6:
-            num_rolls = 6
-        elif num_rolls < 1:
-            output = ":no_entry: You must roll at least once."
-        for _ in range(num_rolls):
-            dice = f"dice_{random.randint(1, 6)}"
-            output += getattr(Emojis, dice, '')
-        await ctx.send(output)
+        def _get_random_die() -> str:
+            """Generate a random die emoji, ready to be sent on Discord."""
+            die_name = f"dice_{random.randint(1, 6)}"
+            return getattr(Emojis, die_name)
```

Comment:
We don't want to create this function every time the roll command is used, so move it up into the cog.